### PR TITLE
updated nodejs section to install node using nvm instead of apt

### PR DESCRIPTION
**Pull request by dinophile**
What it says in the title. Woo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,7 +237,6 @@ We're going to use rbenv to install Ruby for us. But first, there is more housek
 ```bash
 sudo apt-get install libssl-dev
 sudo apt-get install libsqlite3-dev
-sudo apt-get install build-essential
```
Comment (NatalieBlack) [inline README.md:240]: Why remove this?

File: README.md [hunk 2]
```diff
@@ -284,13 +283,87 @@ rbenv global version_number_goes_here
...
+Then we'll add nvm using its native installation script (where we have v0.32.2 make sure you're including the most up to date version of nvm. Check for the latest release candidate [here.](https://github.com/creationix/nvm/releases):
+
+```bash
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.2/install.sh | bash
+```
+
+Then refresh our new nvm settings:
+
+```bash
+. ~/.profile
```
Comment (NatalieBlack) [inline README.md]: Should this be ~/.bashrc?

File: README.md (hunk 1 above)
Comment (dinophile) [reply README.md:240]: Oh they added this back in as a default dependency with a random software update for 16.04, and if they install 17.04 it's also there. It wasn't included originally in 16.04! Problem solved!

File: README.md (hunk 2 above)
Comment (dinophile) [reply README.md]: Oops. Yes. Inserting my own foibles in there...

**Pull request merged**